tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_edit_export_contact.py

```diff
@@ -68,6 +68,8 @@ def test_import_edit_export_contact(self):
         self.assertEqual(self.sim_contact['tel'][0]['value'], self.contact.tel['value'])
 
     def tearDown(self):
-        self.data_layer.delete_sim_contact(self.sim_contact['id'])
+        for contact in self.data_layer.sim_contacts:
+            if contact['name'][0].startswith('Name'):
+                self.data_layer.delete_sim_contact(contact['id'])
```

Comment:
This should be safe, but wouldn't it be better if we create a contact name with 'GAIAUITEST_' prefix?  then it's almost guaranteed that no one would use such prefix, and when the user later encounters this contact on the sim card (by any chance the deletion didn't work) they'll know right away where this contact came from.